README.md

```diff
@@ -24,6 +24,21 @@ cf set-env $YOUR_APP_NAME DD_API_KEY $YOUR_DATADOG_API_KEY
 cf restage $YOUR_APP_NAME
 ```
 
+You can also enable log collection, you will need to provide a TCP port to forward logs to the agent.
+By default, it collects all logs on stdout and stderr, you can disable this functionality if you just want your application logs,
+you will need to forward your logs to the agent using TCP forwarding.
+```shell
+# set the environment variables
+cf set-env $YOUR_APP_NAME DD_API_KEY $YOUR_DATADOG_API_KEY
+cf set-env $YOUR_APP_NAME RUN_PUPPY true
+cf set-env $YOUR_APP_NAME DD_LOGS_ENABLED true
+cf set-env $YOUR_APP_NAME DD_LOGS_CONFIG_TCP_FORWARD_PORT 10514
```

Comment:
The purpose of this option might be a bit confusing for the users. I think we should inform them that they just need to put an unused TCP port

Reply:
it's always used, see https://github.com/DataDog/datadog-cloudfoundry-buildpack/blob/b22bdb5d343666babf260ef9b0d819d8e3ad22cf/lib/redirect-logs.sh#L5

Reply:
Yes I get that, I meant as someone that only want to forward the logs from my std(out|err) "why do I have to configure a port ?"

Reply:
But that was just a thought, feel free to ignore ;)

Reply:
That's totally valid, we could set a default value.

Reply:
Updated